### DISCUSSION TITLE
Domain transfer

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#3968] [Help]
### Summary

Attempting to transfer Vercel domain.

Moved vercel domain, "example-slbt.app". From vercel account A to vercel account B. Deleted the project using this domain on vercel account A.

Created new vercel project on vercel account B. Went to Project settings -> Domains, Edited domain name to moved domain name "example-slbt.app".

Get error "Invalid Configuration".
The error helper states:
![image](https://github.com/vercel/community/assets/142452713/6b1d5be5-ad81-4981-aebe-d6b52ea21446)

But this DNS record already exists for the domain, as they are DNS records created by Vercel and can't be edited.

Note to mention that this domain was working previously, prior to transfer. Any help appreciated.

### Steps to Reproduce

- Create project on vercel account A
- Transfer domain of project A to account B
- Delete project associated with transfered domain on vercel account A
- Create project on vercel account B
- Edit domain of project to transfered domain on vercel account B

**Comment by amyegan** [comment]
Hey @Jammors. When [transferring domains to another team or project](https://vercel.com/docs/projects/domains/transfer-your-domain), you should receive an email asking you to confirm the transfer. After accepting the transfer and adding the domain to your project, you should see it on your account's [Domains page](https://vercel.com/dashboard/domains) and in project settings on the Domains tab.

If you already did all of that and still can't get the domain to work, please let me know. If you share the domain I can look for more info 🙏
